core/src/main/scala/org/apache/spark/sql/hive/HiveClientUtil.scala

```diff
@@ -272,8 +260,10 @@ class HiveClientUtil(val sparkContext: SparkContext) extends Logging {
     val sc = sparkContext
     Property.MetaStoreDBURL.getOption(sparkConf) match {
       case Some(url) =>
-        val driver = Property.MetaStoreDriver.getOption(sparkConf).orNull
-        (false, url, driver)
+//        val driver = Property.MetaStoreDriver.getOption(sparkConf).orNull
+//        (false, url, driver)
+        throw new IllegalArgumentException(s"Invalid property ${Property.MetaStoreDBURL}. " +
+            s"SnappyData does not allow use of property ${Property.MetaStoreDBURL}")
```

Comment:
Done. Removed properties MetaStoreDBURL and MetaStoreDriver from Literals.scala. Also changed the method as per the comment. Filing a separate bug to remove obsolete cluster modes (External and ExternalEmbedded)